packages/rules-engine/src/index.ts

```diff
@@ -1,11 +1,17 @@
-import { JsonLogicRule } from "./lib/rule-engine";
+import { JmesPathRule, JsonLogicRule } from "./lib/rule-engine";
 import { TCreateRuleEngine } from "./lib/types";
 
 
 export const createRuleEngine: TCreateRuleEngine = (options) => {
-    return {
-      logicRule: (rule: any) => {
-        return new JsonLogicRule(rule);
-      }
+  if (options.Provider === 'jmespath') return {
+    logicRule: (rule: any) => {
+      return new JmesPathRule(rule);
     }
+  }
```

Comment:
Not very readable at the moment. Wrapping with curly brackets will help.
```suggestion
  if (options.Provider === 'jmespath') {
    return {
      logicRule: (rule: any) => {
        return new JmesPathRule(rule);
      }
    }
  }
```